MaxText/decode.py

```diff
@@ -29,7 +29,7 @@
 
 import pyconfig
 import max_utils
-from input_pipeline import create_data_iterator_with_tokenizer
+from input_pipeline.input_pipeline import create_data_iterator_with_tokenizer
```

Comment:
nit: input_pipeline/input_pipeline.py is a slightly sad name but I htink it is fine.

Reply:
I'm open to a name suggestion!

Reply:
Are we designing the system so that input_pipeline/input_pipeline.py is the only interface? If so input_pipeline_interface.py is a helpful name (and hiding the other names as prefaced with an underscore is best).

Reply:
done